### ISSUE TITLE
Searching for Music Artist ID (MusicBrainz)

### ISSUE THREAD

**Issue by hasezoey** [jellyfin/jellyfin#6692]
**Describe the bug**
Currently when trying to identify with musicbrainz (in this case a artist by musicbrainz id) shows empty results, but after reload the id is applied

**System (please complete the following information):**
 - OS: Ubuntu Server 21.04 (Jellyfin server), Manjaro 21.1.4 Firefox (Client)
 - Virtualization: Docker
 - Clients: Browser
 - Browser: Firefox 93
 - Jellyfin Version: Jellyfin Docker 10.8.0 (`jellyfin/jellyfin:unstable@sha256:c2bd230f4425e8bb7f95f6f9a18f1aa9589ffa062173c47a871da524e16c3154`)
 - FFmpeg Version: Whatever is in the docker by default
 - Playback: Not applicable to this issue
 - Hardware Acceleration: Not applicable to this issue
 - Installed Plugins: MusicBrainz
 - Reverse Proxy: nginx
 - Base URL: yes, Not applicable to this issue
 - Networking: Docker (Host)
 - Storage: Local (Docker bind mount)

**To Reproduce**
1. have an unmatched artist
2. find a musicbrainz artist id
3. click on `Identify` under `More`
4. enter a artist id
5. click `Search` or press `Enter`

**Expected behavior**
Either:
- to show a result when the id is valid and found in musicbrainz and only apply the id after clicking on it, or
- after entering the id, then clicking `Search` or `Enter` to close the dialog when the id is valid

also, it should not be needed to refresh the page to show that the metadata has been applied

**Server Logs**

**Browser Console Logs**

**Screenshots**

**Additional context**

I post this here in jellyfin issues, because i am pretty sure this does not just apply to the musicbrainz plugin

**Comment by hasezoey** [comment]
with ` sha256:d8b31c242627ffc043c2d579b4f1b81394bb3ead4ce8737a80c7ea31983890e9 ` (docker), i can confirm this is still a issue

**Comment by sonostrass** [comment]
I confirm it's doesn't work either for Musicbrainz nor AudioDB

**Comment by Sparkrdom** [comment]
Still an issue today

**Comment by FirelightFlagboy** [comment]
No stale

**Comment by snex** [comment]
I am experiencing this issue as well. Can somebody please actually look at this instead of hoping your GitHub bot just automatically closes it?

**Comment by felix920506** [comment]
Please understand that Jellyfin is run by a bunch of volunteers, and people might have things to do in their lives that aren't working on Jellyfin bugs. This one unfortunately is going to be a low priority items as there are other more severe issues waiting to be fixed. If you would like to see this issue addressed, a good way would be to fix it yourself and submit a pull request.

**Comment by snex** [comment]
This is a very poor excuse to not at the very least *acknowledge* the bug for over 2 years and to just let it auto-close. In fact, I see an auto-closed issue with the same problem from even longer ago.

And it's also a very poor way to address your users who show understandable frustration at a core feature not working and getting zero feedback about it when reporting it. You should be grateful that people like your work enough to report bugs. Reporting bugs IS contributing. Not everyone knows how to code or has the free time to be able to do it. For all you know, the person reporting this bug is a core maintainer of the software stack YOU use to write Jellyfin, so try to show a little more respect, yeah?

**Comment by JPVenson** [comment]
> For all you know, the person reporting this bug is a core maintainer

If that would be the case, that person would either fix it themself or know that there is no requirement for anyone to fix anything. Everyone including the core team can fix as they like or dont like. 

You are entitled to nothing. 

We have that bot because otherwise we would overflow with both obsolete bugs and ones that are very minor. This is the latter. This breaks nothing and there is a workaround. If somebody would diverge time to work on this other more pressing issues that actually effects the usage of JF, would stay on the line and imagine the outcry then. This project is just grown way out and with the few people who actually fix stuff you should be grateful that JF still exists. We are still in very much need of more contributors.

**Comment by snex** [comment]
> If that would be the case, that person would either fix it themself 

Try reading the entire post. I said "core maintainer of the software stack YOU use to write Jellyfin." In other words, somebody who spends all day dealing with YOUR error reports and doesn't complain about you being "entitled."

By the way, nobody is saying they are "entitled" to anything, although we absolutely are entitled to some basic respect, which you clearly lack. If the project is too much for you to handle, leave and do something else. Don't whine about "entitled users." Your behavior here is WHY you probably don't have enough resources. Who wants to work with somebody that hates their own userbase? Sheesh. Get a clue dude. You're doing nothing but driving people away.

Next time, a simple "sorry, it slipped through the cracks" will do.

**Comment by JPVenson** [comment]
> Next time, a simple "sorry, it slipped through the cracks" will do.

And that is exactly the attitude that is the root of the issue here.
No it did not.
Nobody cared enough to fix it.
You complain that it does not get fixed while not understanding that you have no right (thats the entitlement part) for anything to get fixed.

**Comment by snex** [comment]
I didn't complain that it didn't get fixed. I complained that you are outright ignoring it.

**Comment by JPVenson** [comment]
>  I complained that you are outright ignoring it.

> Nobody cared enough to fix it.

Yes because we (the overall team) cannot force any one to work on anything. Welcome to FOSS. 
If you really want something done, do it yourself.

**Comment by snex** [comment]
Reporting bugs IS doing something.

Next time you need some food, don't go to the grocery store. Grow it yourself.

**Comment by oddstr13** [comment]
> This is a very poor excuse to not at the very least _acknowledge_ the bug for over 2 years and to just let it auto-close. In fact, I see an auto-closed issue with the same problem from even longer ago.

The fact that an issue is left open after triage (sorting) is in and of itself an acknowledgement that this is indeed  an issue that should be fixed, or at least looked at closer by someone familiar with how it's supposed to work.

The reason it hasn't been fixed yet is likely a combination of two things;
1. None of the current code contributors are encountering the issue *or* are sufficiently annoyed by it.
2. The issue doesn't contain enough information to *easily* pinpoint where the issue actually comes from, and how to fix it.

If you're wondering how I personally prioritize issues to work on, it's something along these lines;
1. Security issue; user's personal data is at risk from external actors.
2. I can't watch what I want to watch.
3. Easy fix / interesting issue, and I'm in the mood for working *on Jellyfin* (I have many other hobby projects beside Jellyfin, most are not publicly visible.)
4. It annoys me personally.

This particular issue doesn't meet the bar for any of those for me (maybe "interesting issue", but I don't currently have the bandwidth for it).

> Next time you need some food, don't go to the grocery store. Grow it yourself.

Thanks, I will – currently working on preparing the greenhouse and pre-cultivation area, and ordering things I'll need for the growing season. This has much higher priority than Jellyfin. Potatoes are delicious, nothing beats home-grown sun-ripened tomatoes, and the choice in different food crops and species is far superior to what's available at the supermarket.

---

My suggestion to you is to upvote the the issue – maybe someone are bored one day and decide to sort issues by most upvoted and pick one of the top ones as their entertainment for the weekend.
Alternatively if the problem really bothers you, I would highly recommend you learn enough programming to figure out what's going wrong. The primary strength of Open Source software, as I see it, is that anyone can dive into the code and fix the issues that bother them the most and contribute that back to the project to benefit all other users. The fact that you don't currently know how to code does not prevent this, an issue that really bother you is a great motivator for learning.

**Comment by FirelightFlagboy** [comment]
Not stale

**Comment by FirelightFlagboy** [comment]
not stale

**Comment by FirelightFlagboy** [comment]
not stale

**Comment by hasezoey** [comment]
I dont think this is a problem with jellyfin(web) `10.10.7` anymore.

Current behavior:
- have a matched *or* unmatched artist
- click on `Identify`
- enter a valid musicbrainz artist id
- click `Search` or press `Enter`
- a list of possible values is shown `*1`
- clicking on one brings one to another window (confirmation window)
- click on `Ok`
- window closes, metadata is applied; no reload required

Previously (if i remember correctly), the problem was that at `*1` in the above list, the results page was empty, but the musicbrainz id was still applied, but only visible after a reload.

I will let it stay open for other users to confirm that it is fixed too. (and i am not just misremembering the steps or behavior)

**Comment by foss-** [comment]
Shouldn't a unique ID of a database only bring up one result?

**Comment by hasezoey** [comment]
> Shouldn't a unique ID of a database only bring up one result?

It did only bring up only one result; at least for the ones i tried. The `a list of possible values is shown` was just because i also tried the other search options and they show more. (they use the same results interface)

**Comment by foss-** [comment]
Confirming in 10.10.7. Guess this can be closed?
A similar issue for theaudiodb sadly is persisting: https://github.com/jellyfin/jellyfin/issues/13980

**Comment by hasezoey** [comment]
I will close this as it has basically been fixed, with no one reporting any new issues with it.
